tests/phpunit/Unit/iCalendar/DateParserTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace SRF\Tests\iCalendar;
+
+use SRF\iCalendar\DateParser;
+
+/**
+ * @covers \SRF\iCalendar\DateParser
+ * @group semantic-result-formats
+ *
+ * @license GNU GPL v2+
+ * @since 3.1
```

Comment:
This class is not package public so this is clutter IMO

Reply:
I'm not a fan of the "no comment" purist approach, so do I think some minimal style block should be retained.

Reply:
I was talking just about the @since tag